static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -37,6 +39,10 @@ const hideLoader = (): void => {
     $(LOADER).addClass(IS_HIDDEN_CLASSNAME);
 };
 
+const displayLoader = (): void => {
+    $(LOADER).addClass(IS_HIDDEN_CLASSNAME);
+};
```

Comment:
dotcom is not looking so no biggie and defs not on this pr but they are trying to get rid of `qwery` and use the polyfilled native API! so something like `document.querySelector(LOADER).classList.add(IS_HIDDEN_CLASSNAME)` would be better for new code.